ch.elexis.core.data/src/ch/elexis/data/Query.java

```diff
@@ -658,7 +658,7 @@ public void orderBy(final boolean reverse, final String... n1){
 			ElexisStatus status =
 				new ElexisStatus(ElexisStatus.ERROR, CoreHub.PLUGIN_ID, ElexisStatus.CODE_NONE,
 					"Fehler bei Datenbankabfrage " + ex.getMessage(), ex, ElexisStatus.LOG_ERRORS);
-			log.warn("Fehler bei Datenbankabfrage: " + ex.getMessage());
+			log.warn(String.format("Fehler %s bei Datenbankabfrage:\n%s;", ex.getMessage(), expr));
```

Comment:
You could try the slf4j log native expression
```
log.warn(Error [{}] in database query: [{}]", ex.getMessage, expr);
```
log4j automatically replaces {} expressions with the resp. parameter. Additionally I prefer to put error content within square brackets for better differentiation and copy/paste selection :)